r2-shared/Publication/Publication.ts

```diff
@@ -0,0 +1,93 @@
+import Format from "../Format/Format";
+import Metadata from "./Metadata";
+import PublicationManifest from "./PublicationManifest";
+import { Link, Links } from "./Link";
+
+export default class Publication {
+  public manifest: PublicationManifest;
+  public fetcher: any | null = null; // tmp
+  public services: any | null = null; // tmp
+  public format: Format | null = null;
+  public formatVersion: string | null = null;
```

Comment:
We are not exposing `formatVersion` anymore (it will be soon deprecated on mobile), and I think we won't be adding `format` as well. I think it's something we need to discuss more on a Readium call.